src/components/button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Button = () => {
+    return(
+        <div>
+            <button style={{
+                backgroundColor:"white",
+                fontColor:"blue",
+                borderColor:"white",
+                borderRadius:"10"
+            }}
+            onClick={()=>console.log(name, email)}>Sign up</button>
```

Comment:
This should take a prop so that when we want to use the button on another page we can